couchbase/admin/bucket/v1/bucket.proto

```diff
@@ -64,6 +64,9 @@ message ListBucketsResponse {
     optional couchbase.kv.v1.DurabilityLevel minimum_durability_level = 10;
     optional StorageBackend storage_backend = 11;
     ConflictResolutionType conflict_resolution_type = 12;
+    optional bool history_retention_enabled = 13;
```

Comment:
Is there a reason that we aren't using `history_retention_collection_default`. Otherwise we'll get somewhat confusing lines in gocb:
```
request.HistoryRetentionEnabled = settings.HistoryRetentionCollectionDefault
```